.augment/commands/bmad/workflows/bmm-tech-spec-sm.md

```
@@ -0,0 +1,51 @@
+# Templates
+user_story_template: "{installed_path}/user-story-template.md"
+epics_template: "{installed_path}/epics-template.md"
+
+# Output configuration
+default_output_file: "{output_folder}/tech-spec.md"
+user_story_file: "{output_folder}/user-story.md"
+epics_file: "{output_folder}/epics.md"
+
+# Recommended input documents (optional for Level 0)
+recommended_inputs:
+  - bug_report: "Bug description or issue ticket"
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)